packages/nextjs/components/onchain-impact-dashboard/projectTotalsGraph/projectTotalsGraph.tsx

```diff
@@ -32,41 +33,48 @@ export const ProjectTotalsGraph = ({
   totalsRecord,
   filter,
 }: IProps) => {
-  const formatNumber = (num: number, dec?: number) => {
-    const m = 1000000;
-    const k = 1000;
-    const text = num >= m ? "M" : num >= k ? "K" : "";
-    if (dec != undefined) {
-      const divisor = num >= m ? m : num >= k ? k : 1;
-      const value = new Intl.NumberFormat().format(parseFloat((num / divisor).toFixed(dec)));
-      return `${value}${text}`;
-    }
-    return new Intl.NumberFormat().format(num);
-  };
-  const CustomTooltip = ({ active, payload, label }: any) => {
-    if (active && payload && payload.length) {
-      return (
-        <div className="bg-base-300 p-4 rounded-lg shadow">
-          <p className="m-0 p-0 label font-bold">{formatDate(label)}</p>
-          {payload.map((entry: any, index: any) => (
-            <div key={`item-${index}`}>
-              <p style={{ color: entry.color }} className="m-0 p-0 label">
-                {formatNumber(entry.value)}
-              </p>
-            </div>
-          ))}
-        </div>
-      );
-    }
-    return null;
+  const options = {
+    chart: {
+      type: "line",
+      height: 360,
+    },
+    title: {
+      text: "",
+    },
+    subtitle: {
+      text: "",
+    },
+    credits: {
+      enabled: false,
+    },
+    xAxis: {
+      type: "datetime",
+      title: {
+        text: "",
+      },
+      dateTimeLabelFormats: {
+        day: "%e %b",
```

Comment:
This feels cluttered to me. Currently only showing every other day, could we make it show even less? 
![image](https://github.com/user-attachments/assets/0fee416d-3988-4f21-93f3-2435bee1b488)